sdk/formrecognizer/azure-ai-formrecognizer/tests/test_resolve_elements.py

```diff
@@ -0,0 +1,50 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import re
+from azure.ai.formrecognizer._helpers import get_element_type
+from testcase import FormRecognizerTest
+
+
+class TestResolveElements(FormRecognizerTest):
+
+    def test_word_reference(self):
+        # unlabeled
+        word_unlabeled_ref = "#/readResults/13/lines/91/words/1000"
+        assert get_element_type(word_unlabeled_ref) == "word"
+        indices = [int(s) for s in re.findall(r"\d+", word_unlabeled_ref)]
+        assert [13, 91, 1000] == indices
+
+        # labeled
+        word_labeled_ref = "#/analyzeResult/readResults/0/lines/46/words/0"
```

Comment:
can you also add a test where there's a number after `analyzeResult`

Reply:
discussed offline